hw-11-weatherForecast/Location.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+public class Location
+{
+    public Main main { get; set; }
```

Comment:
https://github.com/ksrvv/hw-10-finance/pull/1#discussion_r972009867

```suggestion
   [JsonPropertyName("main")]
    public Main Main { get; set; }
```